components/DeductionLists/DeductionLists.module.css

```
@@ -1,19 +1,42 @@
 .deductionSumList {
-  display: grid;
-  grid-template-columns: repeat(4, 1fr);
-  justify-items: center;
+  display: block;
   gap: 20px;
 
+  padding: 0 0;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)